iOS/Shimmer/component/component.yaml

```diff
@@ -16,8 +16,12 @@ libraries:
   ios-unified: ../output/unified/Shimmer.dll
 
 samples:
-  - name: "Storyboard Sample"
+  - name: "Shimmer Sample"
     path: ../samples/ShimmerSample/ShimmerSample.sln
-    configuration: "Release|iPhone"
+    installNuGets:
+      - project: ShimmerSample
+        packages:
+          - Xamarin.Shimmer
+    configuration: "Release|iPhone"    
 
 monodoc: ../docs/
```

Comment:
You need to add the externals file to the yaml output.  

https://github.com/xamarin/XamarinComponents/pull/311/files#diff-a928b56fc399e0e478f1ed32a6fd5ab6R33